composer.json

```diff
@@ -41,7 +41,8 @@
     "install-codestandards": [
       "Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run"
     ],
-    "cs-check": "vendor/bin/phpcs",
+    "phpcs": "vendor/bin/phpcs",
```

Comment:
By all means keep the individual scripts, but I'd rather see a script that ran both of these PHPCS together, otherwise it's going to be easy to miss one or the other.
```json
"phpcs-src": "vendor/bin/phpcs",
"phpcs-tests": "vendor/bin/phpcs --runtime-set testVersion 5.6 tests/phpunit/",
"phpcs": [
  "@phpcs-src",
  "@phpcs-tests"
],
```

Now all code can be checked with `composer phpcs`.